ipi_tests/regression_tests/README

```diff
@@ -1,15 +1,33 @@
-See documentaion of how to run the regression tests by running:
+----------------------------------------------------------------------------------
+                   R E G R E S S I O N   T E S T S
+----------------------------------------------------------------------------------
+
+INFORMATION ON CREATING NEW REGRESSION TESTS
+----------------------------------------------------------------------------------
+
+See the documentaion of how to run the regression tests by running:
 
    "python test_run.py -h"
 
 To add a new regression test please provide:
 
    - input.xml (and all the files required by it)
 
-   - driver.txt specifing  "<driver> <address> <port> <unix/inet> " (see existing examples)
+   - driver.txt specifying  "<driver> <address> <port> <unix/inet> " (see existing examples)
+	you should run the driver with the command
```

Comment:
what do you mean? 
Shoudn't be something like " the driver will be called using the following syntax ..."

Reply:
Yes, you are totally right, I got a little confused with the wording and was addressing the creation of the references.